pkg/node/node.go

```diff
@@ -602,6 +649,9 @@ func (s *CSINodeService) NodeUnpublishVolume(ctx context.Context, req *csi.NodeU
 	volumeCR.Spec.Owners = owners
 	if len(volumeCR.Spec.Owners) == 0 {
 		volumeCR.Spec.CSIStatus = apiV1.VolumeReady
+	} else {
```

Comment:
it seems this is not needed,  the default state is Published in this function.

Reply:
because we will proceed on the failed volume now, here the reset of status to Published is for Failed volume successfully unpublished and used by multiple pods.